lib/SRV/Prolog.pm

```diff
@@ -96,9 +97,13 @@ sub cleanup_context {
 sub setup_context {
     my ( $self, $env ) = @_;
 
+    my $start = time();
     my $C = new Context;
     my $req = Plack::Request->new($env);
 
+    my $metrics = $env->{'psgix.metrics'};
+    $C->set_object('Metrics', $metrics) if $metrics;
```

Comment:
If we use a derivative of `HTFeed::JobMetrics` as a singleton, then we don't need to stuff this in the environment/context